sdk/monitor/azure-monitor-ingestion/src/main/java/com/azure/monitor/ingestion/LogsIngestionAsyncClient.java

```diff
@@ -116,7 +156,17 @@ public Mono<Void> upload(String ruleId, String streamName,
     }
 
     /**
-     * See error response code and error response message for more detail.
+     * This method is used to upload logs to Azure Monitor with specified data collection rule id and stream name. This
```

Comment:
I suggest we say "Log Analytics", so that we don't imply support for uploading metrics. Azure Monitor is the umbrella term for a collection of services.
```suggestion
     * This method is used to upload logs to Log Analytics with specified data collection rule ID and stream name. This
```